### PR TITLE
[REVIEW] Add binary columns to fill operations

### LINKED ISSUE #387: Fill Missing Op option to create a new binary column indicating the value was replaced.
Fill Missing Op should have the option to create a new binary column indicating whether the column was filled or not for continuous variables.

This is a common feature used when dealing with missing values of categoricals.

### PR DESCRIPTION
**Pull request by karlhigley**
This PR adds an option to the `FillMissing` and `FillMedian` ops to add an additional binary column that captures whether or not each cell in the original column was filled.

Resolves #387.

### CODE REVIEW

**Comment by karlhigley** [comment]
rerun tests

**Comment by karlhigley** [comment]
Tests are passing locally but failing on CI. 🤔 

(Thought maybe I'd failed to commit or push changes, but looks like they're all here.)

File: nvtabular/ops/fill.py [hunk 1]
```diff
@@ -38,16 +38,30 @@ class FillMissing(Operator):
         The constant value to replace missing values with.
     """
 
-    def __init__(self, fill_val=0):
+    def __init__(self, fill_val=0, add_binary_cols=False):
```
Comment (benfred) [inline nvtabular/ops/fill.py:41]: Can you add a brief description for the ```add_binary_cols``` parameter to the docstring?

**Review by benfred — COMMENTED** [review]
Looks good!

**Review by benfred — APPROVED** [review]

**Pull request merged**